Frameworks/UIKit/NSParagraphStyle.mm

```diff
@@ -38,42 +43,95 @@ + (NSWritingDirection)defaultWritingDirectionForLanguage:(NSString*)languageName
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (instancetype)copyWithZone:(NSZone*)zone {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self retain];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (instancetype)mutableCopyWithZone:(NSZone*)zone {
-    UNIMPLEMENTED();
-    return StubReturn();
+    NSMutableParagraphStyle* copy = [[NSMutableParagraphStyle alloc] init];
+    [copy setParagraphStyle:self];
+    return copy;
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (instancetype)initWithCoder:(NSCoder*)decoder {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (self = [super init]) {
+        _alignment = [[decoder decodeObjectForKey:@"_NSParagraphStyle._alignment"] unsignedIntegerValue];
+        _firstLineHeadIndent = [decoder decodeFloatForKey:@"_NSParagraphStyle._firstLineHeadIndent"];
+        _headIndent = [decoder decodeFloatForKey:@"_NSParagraphStyle._headIndent"];
+        _tailIndent = [decoder decodeFloatForKey:@"_NSParagraphStyle._tailIndent"];
+        _lineBreakMode = [[decoder decodeObjectForKey:@"_NSParagraphStyle._lineBreakMode"] unsignedIntegerValue];
+        _maximumLineHeight = [decoder decodeFloatForKey:@"_NSParagraphStyle._maximumLineHeight"];
+        _minimumLineHeight = [decoder decodeFloatForKey:@"_NSParagraphStyle._minimumLineHeight"];
+        _lineSpacing = [decoder decodeFloatForKey:@"_NSParagraphStyle._lineSpacing"];
+        _paragraphSpacing = [decoder decodeFloatForKey:@"_NSParagraphStyle._paragraphSpacing"];
+        _paragraphSpacingBefore = [decoder decodeFloatForKey:@"_NSParagraphStyle._paragraphSpacingBefore"];
+        _baseWritingDirection =
+            (NSWritingDirection)[[decoder decodeObjectForKey:@"_NSParagraphStyle._baseWritingDirection"] unsignedIntegerValue];
+        _lineHeightMultiple = [decoder decodeFloatForKey:@"_NSParagraphStyle._lineHeightMultiple"];
+        _tabStops = [decoder decodeObjectForKey:@"_NSParagraphStyle._tabStops"];
```

Comment:
>[decoder decodeObjectForKey:@"_NSParagraphStyle._tabStops"]; [](start = 20, length = 60)

!NO!

If this file is not ARC-enabled, and _tabStops is not a `StrongId` (it's a property's backing ivar, so it's not likely to be a `StrongId`), you must *-retain* it.